container/container.go

```diff
@@ -66,7 +71,7 @@ func New(options ...Option) (*DockerContainer, error) {
 		}
 	}
 	c.negotiateAPIVersion()
-	if err := c.createSwarmIfNeeded(); err != nil {
+	if err := c.checkSwarm(); err != nil {
```

Comment:
Rename s/checkSwarm/ - it checks for what? 

`IsSwarmInit (bool, error)` or as You didt `IsSwarmInit error`

Reply:
what about `checkSwarmInit() error` ?

Reply:
what's wrong with `is`? checkSwarmInit checks but for what?

you can check if is running, is init, is up etc
CheckUp, CheckRunning etc, but check alone doesn't carry any more info.

Reply:
Rename by `isSwarmInit() error`